README.md

```diff
@@ -523,7 +530,7 @@ DABlin implements (at least partly) the following DAB standards:
 
 ## TODO
 
-At the moment, DABlin is kind of a rudimentary tool for the playback of
+At the moment, DABlin is a kind of rudimentary tool for the playback of
```

Comment:
This is already valid English, as far as I can see...

Reply:
Your text says DABlin could be a tool,
mine - on the other hand - DABlin is a tool.

See https://forum.wordreference.com/threads/a-kind-of-kind-of-a.1727570/

Reply:
Well, I guess that in the end it's up to a user to decide whether it *could be* or *is* a tool for him/her. Unless there is a strong reason (e.g. due to a spelling mistake or wrong details), there is no real improvement in changing text.